admin-ui/app/routes/Apps/Gluu/GluuSelectRow.tsx

```diff
@@ -56,48 +56,71 @@ function GluuSelectRow({
     })
   }
 
+  // Calculate grid sizes based on Bootstrap column system (out of 12)
+  const labelGridSize = Math.round((lsize / 12) * 12)
+  const selectGridSize = Math.round((rsize / 12) * 12)
```

Comment:
This grid size calculation is bad because the logic is fundamentally flawed and redundant; the expression `Math.round((lsize / 12) * 12)` will always resolve to `lsize`, adding unnecessary complexity without any functional benefit.)